TailLight/driver.h

```diff
@@ -12,9 +12,15 @@
 #include "wmi.h"
 #include "vfeature.h"
 
-
 /** Memory allocation tag name (for debugging leaks). */
-static constexpr ULONG POOL_TAG = 'ffly';
+static constexpr ULONG POOL_TAG = 'ylff';
+static constexpr ULONG WDF_POOL_TAG = 'wTLT'; // Taillight WPF allocated
```

Comment:
Comments:
* Please don't bundle changes to `POOL_TAG` with the other changes in this PR. Submit it as a separate PR if you want to change it.
* Why are you introducing `WDF_POOL_TAG`? It doesn't seem to be used anywhere?

Reply:
Done. I now know the rules. WDF_POOL_TAG is to be used in a future checkin so that WDF allocations on behalf of TailLight will be traceable to taillight.. Otherwide, a generic pool tag is used that could refer to any WDF driver.

Reply:
Sorry, but the new `POOL_TAG` define is still in your PR, so the review comment is not yet "done".